src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/CollapsedTab.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React from 'react';
+import collapsedTabStyles from './collapsedTab.scss';
+
+type Props = {
+    children: string,
+    index: number,
+    onClick: (index: number) => void,
+};
+
+export default class CollapsedTab extends React.PureComponent<Props> {
```

Comment:
The `TabComponent` passes it's width up to the `TabsComponent` initially and when it changes it's size. So with the current solution, if a tab collapses, it's removed from the dom and therefore it's width doesn't change. With your suggestion, the tab would be rendered in the list for the collapsed tabs after collapsing (with a width of 100% of parent), and therefore pass a new width to the `Tabs` component and that would break further calculations in the `TabsComponent`